docs/UserGuide.md

```diff
@@ -285,26 +286,26 @@ The index **must be a positive integer** 1, 2, 3, …​
 
 ### Finding tasks: `findt`
 
-Find tasks whose task names contain any of the given keywords and/or by tags.
+Find tasks whose task names and/or tags contain any of the given keywords.
 
 **Format**: `findt KEYWORD [MORE_KEYWORDS]...[TAG]...`
-
-* The search is case-insensitive. e.g `eat` will match `Eat`
-* The order of the keywords does not matter. e.g. `Eat Apple` will match `Apple Eat`
-* Only the name is searched.
-* Only full words will be matched e.g. `Apples` will not match `Apple`
+* `KEYWORD` can be used to search for matching words in both tags and names of tasks.
+* The search is case-insensitive. e.g `lecture` will match `Lecture`
+* The order of the keywords does not matter. e.g. `Lecture CS2103T` will match `CS2103T Lecture`
+* Only full words will be matched e.g. `Tut` will not match `Tutorial`
 * Tasks matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Apple Pear` will return `Eat apple`, `Buy pear`
+  e.g. `Lecture Tutorial` will return `CS2103T Lecture`, `Tutorial 12`
 
 **Examples**:
-* `findt with` returns `Consultation with students` and `Meeting with invigilators`
-* `findt TAs lecture` returns `Meeting with TAs` and all recurrences of `CS2103T lecture` <br>
+* `findt with` returns `Consultation with students` and `Meeting with invigilators`.
+* `findt week` returns all recurring instances of `CS2103T Lecture` as their tags contain `Week XX Lecture`.
+* `findt TAs lecture` returns `Meeting with TAs` and all recurrences of `CS2103T lecture`. <br>
     <br>
     ![`findt TAs lecture`](images/findtCommandShowcase.png)
 
 <div markdown="span" class="alert alert-info">
 :bulb: **Tip:**
-Similar to [`findc`](#finding-contacts-findc), you can find based on tags too!
+If you have many tasks, properly using `findt` will help you to keep organized! 
```

Comment:
```suggestion
If you have many tasks, using `findt` will help you to keep organized! 
```
I think "properly" is not needed because we can assume that the users will not make any mistakes